nuxeo-runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/connect/update/standalone/PackagePersistence.java

```diff
@@ -157,7 +157,7 @@ public synchronized LocalPackage addPackage(File file) throws PackageException {
             } catch (IOException e) {
                 throw new PackageException("Failed to unzip package: " + file.getName());
             } finally {
-                // cleanup if tmp still exists (should not happen)
+                // cleanup tmp if exists
```

Comment:
You can remove the comment: tmp will always exist since your following change on `addPackageFromDir()`